instrumentation/opentelemetry_ecto/lib/opentelemetry_ecto.ex

```diff
@@ -70,11 +71,12 @@ defmodule OpentelemetryEcto do
     # TODO: need connection information to complete the required attributes
     # net.peer.name or net.peer.ip and net.peer.port
     base_attributes = %{
-      "db.type": db_type,
+      "db.connection_string": url,
```

Comment:
This can expose sensitive credentials so that has to be addressed if you're looking to add it. That is why I didn't add it in the last pass and I don't have an easy solution to address it. :/

Reply:
Yea, should either be optional or removed.